### DISCUSSION TITLE
Pretty format without a terminal

### DISCUSSION THREAD

**Discussion by jeff-cook** [bats-core/bats-core#1045] [Q&A] [answered]
Does anyone know what a can add to my CI job that will allow the Pretty format without a terminal?

While I don't have a terminal my CI (Bitbucket) does display colors. Having the pretty format will make it easier for our staff to read. 

Thanks

**Comment by jeff-cook** [comment]
I set `TERM=xterm-256color` which is that I have that I was working from locally. 

It no longer fails, but the output is just not the same. 
- The red color for failed test don't include test description.
- The checkmark or x is not at the start of the test description, but at the end.

**Reply by martin-schulze-vireso** [reply, marked as answer]
The pretty formatter does not only create colors but also has an interactive update by first printing the test as executing and then overwriting the line with the test result. I assume that your environment is not reacting as intended to the cursor contral ansi escape controls.

**Reply by jeff-cook** [reply]
Thank you
